src/components/Controls.vue

```diff
@@ -509,8 +509,14 @@ export default {
 		width: 44px;
 		height: 44px;
 
-		&:hover, &:focus {
-			background-color: rgba(127,127,127,0.25) !important;
+		&[data-popper-shown] {
+			background-color: rgb(245, 245, 245);
+			[data-themes=dark] & {
+				background-color: #212121;
+			}
```

Comment:
Can you elaborate what that special styling would be needed for? I couldn't notice it would have any effect when testing out the PR. In general we should avoid any hard coded color values and use available css vars wherever appropriate these days.

Reply:
The difference is that when the popover opens, the filter button doesn't look like it's been pressed.

With styles:
<img width="303" alt="Screen Shot 2023-11-20 at 5 19 34 PM" src="https://github.com/nextcloud/deck/assets/91102425/c31f2fbf-cc2e-40b9-988f-9d0479185f86">

Without styles:
<img width="295" alt="Screen Shot 2023-11-20 at 5 20 06 PM" src="https://github.com/nextcloud/deck/assets/91102425/32998067-2be2-4def-93c3-20814542b934">

Reply:
I refer to Talk when the popover opens, the filter button will be like this

<img width="386" alt="Screen Shot 2023-11-20 at 5 22 32 PM" src="https://github.com/nextcloud/deck/assets/91102425/ddbe7081-8f5d-4b25-a2ff-96a6ccde319f">

Reply:
After checking the existing color vars, the required style is just like this

<img width="437" alt="Screen Shot 2023-11-20 at 5 56 00 PM" src="https://github.com/nextcloud/deck/assets/91102425/878b34a1-5c76-4b24-acaa-bf69ea832a5e">

Reply:
Ah good point then, especially with those variables that seems good

Reply:
Thanks, I will update the code